REBASE.openshift.md

```diff
@@ -206,36 +231,24 @@ should be true:
 - [ ] `make` executes without error
 - [ ] `make verify` executes without error
 - [ ] `make test` executes without error
-- [ ] Upstream tags are pushed to `openshift/kubernetes` to ensure that
+- [ ] The upstream tag is pushed to `openshift/kubernetes` to ensure that
       build artifacts are versioned correctly
       - Upstream tooling uses the value of the most recent tag (e.g. `v1.20.0`)
         in the branch history as the version of the binaries it builds.
+      - Pushing the tag is easy as
+```
+git push git@github.com:openshift/kubernetes.git refs/tags/v1.20.0
+```
 
 Details to include in the description of the PR:
 
 - [ ] A link to the rebase spreadsheet for the benefit for reviewers
-- [ ] A comment reminding reviewers of the need for manual upgrade testing
-      along with a `/hold` command to prevent merge until such testing is
-      completed.
-
-In addition to the standard requirement that all CI jobs be passing, the rebase
-PR should not be merged until additional upgrade testing initiated with
-cluster-bot is passing:
-
-- [ ] `test upgrade [previous release e.g. 4.6] openshift/kubernetes#[PR#] [aws|azure]`
-       - Only gcp upgrades are tested automatically via presubmit
-         (`e2e-gcp-upgrade`) and it's necessary to manually test aws and azure.
-- [ ] `test upgrade openshift/kubernetes#[PR#] openshift/kubernetes#[PR#]`
-      - This 'self-upgrade' ensures that it is possible to upgrade _from_ the
-        rebased release. The other upgrade testing validates that it's possible
-        to upgrade _to_ the rebased release.
```

Comment:
why is all this gone? We do that automatically?

Reply:
yep, I've added all of this to CI and it is triggered by changing the kube version in the dockerfile and run automatically. you can see the jobs passing in status

Reply:
e.g. https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_kubernetes/361/pull-ci-openshift-kubernetes-master-e2e-aws-selfupgrade/1320659343942619136